src/components/navBar/index.tsx

```diff
@@ -1,17 +1,94 @@
-import Image from 'next/image';
-import styles from '@/components/navBar/navBar.module.scss';
+import { useEffect, useState } from "react";
+import Image from "next/image";
+import userData from "@/helperFunctions/getUser";
+import { toast, ToastTypes } from "@/helperFunctions/toast";
+import navLinks from "./navLinks";
+import NavbarLinks from "./NavbarLinks";
+import UserSignIn from "./UserSignIn";
+import {
+  RDS_LOGO,
+  HAMBURGER_ICON,
+  SIGN_IN_WITH_GITHUB,
+  SIGN_IN,
+  HOME_PAGE_LINK,
+  TWENTY_FIVE_PX,
+  THIRTY_PX,
+  FIFTY_PX,
+} from "@/components/constants/navbar";
+import styles from "@/components/navBar/navBar.module.scss";
 
-const RDSLogo = '/RDSLogo.png';
+const NavBar = () => {
+  const [toggle, setToggle] = useState(false);
+  const [userIsLoggedIn, setUserIsLoggedIn] = useState(false);
+  const [userDetails, setUserDetails] = useState({
+    userName: "",
+    firstName: "",
+    profilePicture: "",
+  });
 
-const NavBar = () => (
-  <nav className={styles.navBar}>
-    <a className={styles.logo} href="https://realdevsquad.com"><Image width="45px" height="45px" src={RDSLogo} alt="real-dev squad" /></a>
-    <a href="https://welcome.realdevsquad.com/">Welcome</a>
-    <a href="https://www.realdevsquad.com/events.html">Events</a>
-    <a href="https://members.realdevsquad.com/">Members</a>
-    <a href="https://crypto.realdevsquad.com/">Crypto</a>
-    <a className={styles.active} href="https://status.realdevsquad.com/">Status</a>
-  </nav>
-);
+  useEffect(() => {
+    (async () => {
+      await userData()
+        .then((data) => {
+          const { username, first_name, picture } = data;
+          setUserDetails({
+            userName: username,
+            firstName: first_name,
+            profilePicture: picture?.url,
+          });
+
+          setUserIsLoggedIn(true);
+        })
+        .catch((error) => {
+          toast(ToastTypes.ERROR, (error as Error).message);
+        });
+    })();
+  }, [userDetails, userIsLoggedIn]);
+
+  return (
+    <nav className={styles.navBar}>
+      <button className={styles.hamburger} onClick={() => setToggle(!toggle)}>
+        <Image
+          src={HAMBURGER_ICON}
+          alt="Hamburger"
+          height={THIRTY_PX}
```

Comment:
@Neha Ohh ok, in the previous review, I thought you were talking about height and width to make it constant but it was `alt` actually. Will make the change

Reply:
Any suggestion on what should I name this constant as there are different values of height/width at different places

Reply:
I have achieved this in an alternative way. Here instead of using the string, I have used values directly 
![image](https://user-images.githubusercontent.com/78433013/174480047-8bdda094-0c76-4086-8369-d8b6409bf15b.png)